### DISCUSSION TITLE
Syntax Highlighting

### DISCUSSION THREAD

**Discussion by AabyeHald** [cotes2020/jekyll-theme-chirpy#1350] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

Does a list of supported languages for syntax highlighting exist?

It turns out to be a bit trial and error when guessing on what should/can be written after ```

I am particular interested in Bicep, PowerShell, JSON, Azure CLI.

### Operations you have already tried

I have tried out assuming that the Rouge highlighter is the one used, but it seems not to be.
I have used a lot of time so far trying to find out how the syntax highlighting is done, but I think I am running around in circles missing something obvious.

**Comment by cotes2020** [comment, marked as answer]
> I have tried out assuming that the Rouge highlighter is the one used, but it seems not to be.

Why not?

https://github.com/cotes2020/jekyll-theme-chirpy/blob/2bf87e0de7928f325811e1bb96cfcaefdf6cf66a/_config.yml#L120

The most intuitive way is to go to <https://rouge.jneen.net/> and search for supported syntax.

Alternatively, if you have [`rouge-ruby`](https://rubygems.org/gems/rouge) installed locally, you can run the following command in terminal to print a list of all syntaxes:

```console
$ rougify list
```

**Reply by AabyeHald** [reply]
Thanks a lot for the fast response  - as I mentioned it was something obvious I missed - the local install of rouge was missing.
